modules/network/src/main/java/org/apache/ignite/internal/network/scalecube/ScaleCubeTopologyService.java

```diff
@@ -232,4 +272,16 @@ private static NodeMetadata deserializeMetadata(@Nullable ByteBuffer buffer) {
             return null;
         }
     }
+
+    @Override
+    public void onJoined(ClusterNode node) {
+        LOG.info("Node joined logical topology [node={}]", node);
```

Comment:
I'm not sure we need to log it here (and on leave) as INFO. This seems to be useful when debugging this component during development, but in production information about a node being joined the LT is already logged, so this will just add noise.